cmd/snap/cmd_snap_op.go

```diff
@@ -89,11 +86,7 @@ With no further options, the snaps are refreshed to the current revision of the
 channel they're tracking, preserving their confinement options. All available
 channels of a snap are listed in its 'snap info' output.
 
-Revision choice via the --revision override requires the user to
-have developer access to the snap, either directly or through the
-store's collaboration feature, and to be logged in (see 'snap help login').
-
-Note a later refresh will typically undo a revision override.
+When providing --revision note a later refresh will typically undo a revision override.
```

Comment:
```suggestion
When providing --revision note that a later refresh will typically undo a revision override.
```

Reply:
As in the earlier suggestion, I'd tweak this slightly to avoid this looking like providing a `revision note`:

```suggestion
When --revision is used, a later refresh will typically undo the revision override.
```